### ISSUE TITLE
No Braille input for drum kit

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32805]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Currently there does not seem to be any way to trigger input on the Braille panel for a multi instrument untuned percussion staff such as a drum kit.  Neither using the assigned shortcut letters or the letters shown on output (closely related to midi assignment) seem to work with or without octave signs.

Suggest that these should be assigned to the letter name and octave mark they would be if they were written in bass clef, so bottom space A2, top space G3, etc.  This is the most standard method used by transcribers (although the manuals do allow for the transcriber to choose their own mapping).  These need to be fixed in output first (see #32804) so they match for both input and output.  It probably makes sense for them to default to the 1st option of instrument for each of those line/space assignments, and automatically use the correct note head shape on the score without needing to input the note head symbols in Braille.  If these are supported for Braille output they could show when it refreshes.

Input in Braille should follow the standard rules for voices and the use of the in-accord symbol the same as it does in regular staff notation, which already work correctly for output.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

No.

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
